src/core_types.jl

```diff
@@ -97,7 +97,7 @@ end
 Base.@propagate_inbounds Base.getindex(r::RotMatrix, i::Int) = r.mat[i]
 @inline Base.Tuple(r::RotMatrix) = Tuple(r.mat)
 
-@inline RotMatrix(θ::Real) = RotMatrix{2}(θ)
+@inline RotMatrix(θ::Real) = RotMatrix{2}(mod(θ,2pi))
```

Comment:
Technically speaking, `2pi` is a `Float64` but `θ` could be a `BigFloat`, which means this rounding could be inaccurate.

Luckily, Julia has a `mod2pi` function for exactly this, which we should employ here.